sqlglot/dialects/trino.py

```diff
@@ -59,6 +77,11 @@ class Generator(Presto.Generator):
             exp.JSONPathSubscript,
         }
 
+        def jsonextractquote_sql(self, expression: exp.JSONExtractQuote) -> str:
```

Comment:
Move this to the base generator class, otherwise doing `expression.sql()` will crash.